test-requirements.txt

```diff
@@ -1,5 +1,6 @@
 pip>=21.3.1
-pytest==6.0.1
+pytest==6.0.1; python_version < '3.10'
+pytest==7.0.1; python_version == '3.10'
```

Comment:
Am I understanding correctly that we need pytest 7.0.1 when using python 3.10? If so, can we use pytest 7.0.1 for all python versions instead of just for python 3.10?

Reply:
using pytest==7.0.1 for all python versions